root/utility/tableColumns.js

```diff
@@ -533,7 +539,8 @@ export const ratingsColumn:
     Cell: ({row: {original}}) => <RatingStars entity={original} />,
     Header: N_l('Rating'),
     accessor: 'rating',
-    className: 'rating c',
+    cellProps: {className: 'rating c'},
+    headerProps: {className: 'rating c'},
```

Comment:
Didn't we previously only add most of these classes to the headers?

Reply:
Hmm. For ratings specifically, we were super inconsistent. Many didn't have classes at all, but https://github.com/metabrainz/musicbrainz-server/commit/bae4c64c859f9ab752a5e6dca0cab4627689e2c5#diff-bf2f0326c5431ebe0a1f4fc70a8635f5 for example seems to have had 'rating c' on the header and just 'c' on the cell. We could now do that again, if desired.

Reply:
Huh, okay. I'm also pretty sure we didn't have year, barcode-cell, checkbox-cell, actions, and the rest on all the cells either (at least I don't remember seeing these classes on every table cell in the web inspector). If we can avoid adding those unless they're already used or needed, that'd be good. If they're not used by any JS/CSS then they're cluttering the HTML (though existing ones might be used by userscripts).

Reply:
Checked all the commits converting lists to React.
Some had `.pos`: it was used for some checkbox cells but we replaced, is equivalentish to `.checkbox-cell`, and you're right was used only on headers, so deleting `cellProps` for that one.
`.artist` was used on RG lists (only, for some reason?) and also only on headers. Same change.
`.rating` is also on (RG and recording) headers only, but both cells and headers have `.c` on RG (not on recording, because we love consistency). Same for `.count` and `.year` on RG. Changed it back for cells to have just `.c`.
Works used to have `.iswc` on ISWC cells (but not headers). Readding that.
`.barcode-cell` was on cells, not headers. Deleted `headerProps` there.

Reply:
Good catch, changes seem reasonable but I did not review thoroughly. To ease review (and future search through code history), it would have been helpful to split this patch (which is supposed to fix heading ticket mainly) and to add a reference for each of these buggy commits you checked and fixed.